### ISSUE TITLE
Incorporating reverse M–M equation into MEND

### ISSUE THREAD

**Issue by kdorheim** [kdorheim/MENDplus#7]
Hi @jianqiuz as we discussed two weeks ago I am going to try to incorporate reverse MM dynamics into the MEND model. I've been looking at  https://www.sciencedirect.com/science/article/pii/S0038071712003410 to get a better understanding of what is going on. 

Where they define R-MM as 
<img width="201" alt="R-MM" src="https://user-images.githubusercontent.com/27299759/85627529-a4f47a80-b63c-11ea-92c6-69b30d2bc3d7.png">

Is this the representation you would like us to adopt as well? Looking at Wang et al. 2015 the following fluxes between carbon pools are contain the traditional MM kinetics related to biomass growth, `F1` (DOC up take by microbial biomass), `F4` (Microbial respiration from biomass growth) and `F5` (Metabolic/maintenance microbial respiration). Do we want to replace the MM kinetics in all of these fluxes with the R-MM representation or just the two related to biomass growth?

**Comment by jianqiuz** [comment]
Yes, this is the R-MM formulation that we can use to address enzyme saturation. We can approach this in two stages. I think it is reasonable to start with the decomposition of POM1, POM2 and MOM, because these processes are mediated by extracellular enzymes EP1, EP2 and EM. In the second stage, we can evaluate the kinetic formulation of F1, F4 and F5.

**Comment by kdorheim** [comment]
### First Stage Implementation

Here is a plot of MEND set up with default parameters but MOC and POC decomposition is now modeled with R-MM.
![image](https://user-images.githubusercontent.com/27299759/86163629-8bd94700-bade-11ea-9d07-aa5151a68949.png)

To see more details please check out this pdf.
[First_RMM_test.pdf](https://github.com/kdorheim/MENDplus/files/4853793/First_RMM_test.pdf)
[Reproducible script](https://github.com/kdorheim/MENDplus/files/4853801/First_RMM_test.Rmd.zip)
